app/views/course/assessment/submissions/_guided.html.slim

```diff
@@ -0,0 +1,4 @@
+nav
+  ul.pager
```

Comment:
I think pagination makes more sense, especially since there's usually a large number of them. In any case, people who write themes will be confused also.